de/manual/ch_Business.docbook

```diff
@@ -1423,83 +1469,152 @@
-        Verwenden Sie hierzu den Menüpunkt &mc.fi.prin-inv;.
+        Hierzu gibt es 2 unterschiedliche Vorgehensweisen.
       </para>
 
-      <tip>
-        <para>Sie können das Aussehen der Rechnung ändern, wie z.B. ein Firmenlogo hinzufügen usw. Siehe dazu
-          den Abschnitt <xref linkend="busnss-ar-invoicechange" />.
-        </para>
-      </tip>
-
       <procedure>
-        <title>Rechnung als Bericht</title>
+        <title>Rechnung ausdrucken</title>
 
-        <para>Rechnungen können auch vom Hauptfenster gedruckt werden. Im Menü &mc.rep.bus; stehen mehrere
-          Vordrucke zur Auswahl:
-          <itemizedlist>
-            <listitem>
-              <para>&gmi.rep.bus.pnt-rep;
-              </para>
-            </listitem>
+        <step>
+          <para>In der geöffneten Rechnung verwenden Sie hierzu den Menüpunkt &mc.fi.prin-inv;, woraufhin sich ein
+            neuer Reiter die Rechnung in einem Drucklayout öffnet.
+          </para>
 
-            <listitem>
-              <para>&gmi.rep.bus.easy-inv;
-              </para>
-            </listitem>
+          <substeps>
+            <step>
+              <para>Wollen Sie das Erscheinungsbild der Rechnung ändern, betätigen Sie
+                <itemizedlist>
+                  <listitem>
+                    <para>in der <emphasis>Werkzeugleiste</emphasis> die Schaltfläche <guibutton>Optionen</guibutton>
+                    </para>
+                  </listitem>
 
-            <listitem>
-              <para>&gmi.rep.bus.fcy-inv;
+                  <listitem>
+                    <para>oder &mc.ed.rep-opt; aus der Menüleiste.
+                    </para>
+                  </listitem>
+                </itemizedlist>
               </para>
-            </listitem>
+            </step>
 
-            <listitem>
-              <para>&gmi.rep.bus.tax-inv;
+            <step>
+              <para>Aktivieren Sie den Reiter <guilabel>Allgemein</guilabel> in dem Dialog der Berichtsoptionen.
               </para>
-            </listitem>
-          </itemizedlist>
-          Im Berichtsfenster wird angezeigt, dass keine gültige Rechnung ausgewählt wurde. So
-          wählen Sie die zu druckende Rechnung aus:
-        </para>
+            </step>
 
-        <step>
-          <para>Betätigen Sie
-            <itemizedlist>
-              <listitem>
-                <para>in der <emphasis>Werkzeugleiste</emphasis> die Schaltfläche <guibutton>Optionen</guibutton>
-                </para>
-              </listitem>
+            <step>
+              <para>Wählen Sie eine <guilabel>Stilvorlage</guilabel> aus, indem Sie die Schaltfläche
+                <guibutton>Auswählen</guibutton> betätigen. Hier können Sie auf diese
+                vordefinierten Stilvorlagen zurückgreifen.:
+                <itemizedlist>
+                  <listitem>
+                    <para><guilabel>CSS-basierte Stilvorlage (experimentell)</guilabel>
```

Comment:
@jralls @Bob-IT Shouldn't declare CSS based stylesheets after IIRC over 20 years as stable?

Reply:
@fellen, that's up to @christopherlam, he [created](https://github.com/Gnucash/gnucash/commit/85058f82786039a4e4c27d2da2a328df0d876f50) it and designated  it experimental 5 years ago.